docs/modules/operation/pages/performance-data-collection/snmp-index.adoc

```diff
@@ -1,13 +1,13 @@
 
 [[snmp-index]]
-= Custom resource types
+= Create a Custom Resource Type
 
 This section describes how to create a custom resource type to collect SNMP data from tables with arbitrary indexes:
 
```

Comment:
The docs already builds a page-level table of contents on the right. Don't need to have a separate list of page sections here.
```suggestion
This section describes how to create a custom resource type to collect SNMP data from tables with arbitrary indexes.
```